generate.sh

```diff
@@ -0,0 +1,38 @@
+if [ -z "$ASSEMBLY_DATE" ]
+then
+   TIMESTAMP=`eval date +%d.%B.%Y`
+else
+   TIMESTAMP=$ASSEMBLY_DATE
+fi
+
+# Convert abakus LaTeX to html
+pandoc -f latex -t html \
+  --include-in-header=/static/head.html \
+  --include-before-body=/static/abakus.html \
+  --include-after-body=/static/end.html \
+  abakus-statutter/innhold.tex > "/public/index.html"
+
+# Move other static files to the public folder
+cp /static/main.css /static/abakule.jpg /public/
+
+# Convert abakus LaTeX to PDF
+pdflatex -halt-on-error abakus-statutter.tex
+pdflatex -halt-on-error abakus-statutter.tex
+pdflatex -pdf -halt-on-error -output-dir="/public" abakus-statutter.tex
```

Comment:
Why all these steps? At least the two identical ones.

Reply:
https://www.systutorials.com/why-do-i-need-to-run-latex-bibtex-three-times-to-make-everything-look-good/